src/CliConfig.php

```diff
@@ -71,6 +71,21 @@ public function getUserConfigDir()
     }
 
     /**
+     * @param bool $create
+     *
+     * @return string
+     */
+    public function getTempDir($create = true)
+    {
+        $dir = $this->getUserConfigDir() . '/tmp';
+        if (!is_dir($dir) && $create && !mkdir($dir, 0700, true)) {
```

Comment:
I would `!@is_dir($dir)` to avoid any warning. which would seem to appear in case the folder doesn't exist? Same for `!@mkdir` in case of permission issue.